00-Introducing-HOOMD-blue/05-Compressing-the-System.ipynb

```diff
@@ -0,0 +1,442 @@
+{
```

Comment:
<p><code>Tuner</code>s don't change the state do they? The particle sorter changes the particle ordering, but the microstate is still the same.</p><p> </p><p>The class name for the move size tuner is also not the one in the PR (one needs to change).</p><p> </p><p>Also, just a side note, if we use the secant solver for this tutorial with quick compress we will likely need to use a smaller gamma than default because of the dynamic nature of the system.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Yes, I wrote that with the particle sorter in mind. The fact that it changes the system state without changing the microstate of the system is perhaps too fine a point to include in the introductory tutorial. I'll just remove that mention.</p><p> </p><p>I updated the name and do not plan on using the secant solver.</p>